tests/ovirt/basic_vm_import_test.go

```diff
@@ -109,6 +111,61 @@ var _ = Describe("Basic VM import ", func() {
 				},
 			}, tests.StorageClass))
 	})
+
+	Context("when it's successful and the import CR is removed", func() {
+		It("should not affect imported VM or VMI", func() {
+			By("Creating Virtual Machine Import")
+			vmi := utils.VirtualMachineImportCr(vmID, namespace, secret.Name, f.NsPrefix, true)
+			vmImports := f.VMImportClient.V2vV1alpha1().VirtualMachineImports(namespace)
+
+			created, err := vmImports.Create(&vmi)
+
+			Expect(err).NotTo(HaveOccurred())
+			Expect(created).To(BeSuccessful(f))
+
+			retrieved, _ := vmImports.Get(created.Name, metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+
+			vmBlueprint := v1.VirtualMachine{ObjectMeta: metav1.ObjectMeta{Name: retrieved.Status.TargetVMName, Namespace: namespace}}
+			By("Making sure the imported VM is running")
+			Expect(vmBlueprint).To(BeRunning(f))
+
+			When("VM import CR is deleted", func() {
+				foreground := metav1.DeletePropagationForeground
+				deleteOptions := metav1.DeleteOptions{
+					PropagationPolicy: &foreground,
+				}
+				err = vmImports.Delete(retrieved.Name, &deleteOptions)
+				if err != nil {
+					Fail(err.Error())
+				}
+
+				By("Waiting for VM import removal")
+				err = f.EnsureVMIDoesNotExist(retrieved.Name)
```

Comment:
Just a light note, that I am sometimes confused wheter `VMI` is `VirtualMachineInstance` or `VirtualMachineImport`, maybe we should refactor those funtions to be clear what it handle.

Reply:
Good point; I will change it in the next PR.